### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ctf0**
https://github.com/spatie/laravel-permission/issues/358

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,6 +184,22 @@ class User extends Authenticatable
 }
 ```
 
+> - note that if you need to use `HasRoles` trait with another model ex.`Page` you will also need to add `protected $guard_name = 'web';` aswell to that model or you would get an error
```
Comment (freekmurze) [inline README.md]: Small typo: `awell`

Comment (ctf0) [inline README.md]: u mean `as well` ?

Comment (freekmurze) [inline README.md]: yeah, sorry for the typo confusion 😄

Comment (ctf0) [inline README.md]: done 👍

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**